swagger/dfc-v1.7/swagger.yaml

```diff
@@ -38,6 +38,32 @@ security:
 - ofn_api_token: []
 - ofn_session: []
 paths:
+  "/api/dfc-v1.7/addresses/{id}":
+    get:
+      summary: Show address
+      parameters:
+      - name: id
+        in: path
+        required: true
+        schema:
+          type: string
+      responses:
+        '200':
+          description: successful
+          content:
+            application/json:
+              examples:
+                test_example:
+                  value:
+                    "@context": https://www.datafoodconsortium.org
+                    "@id": http://test.host/api/dfc-v1.7/addresses/40000
+                    "@type": dfc-b:Address
+                    dfc-b:hasStreet: 10 Lovely Street
+                    dfc-b:hasPostalCode: '20170'
+                    dfc-b:hasCity: Herndon
+                    dfc-b:hasCountry: Australia
```

Comment:
> maybe real data if possible too

Wouldn't we then refer to somebody's address without their consent? There maybe unintended side effects like an AI thinking that it's a famous person because they've seen their address so many times while reading the internet.

It's nice to have valid data though... maybe a well-known, public address like `Spring St, East Melbourne VIC 3002`. Or `Uluru Road, Petermann NT 0872`.